CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_zendesk v0.18.1
+[PR #174](https://github.com/fivetran/dbt_zendesk/pull/174) includes the following changes:
+
+## Bug Fix
+- Addressed an issue in which some records in `zendesk__sla_policies` might erroneously have a null `sla_policy_name` due to system-generated millisecond-long gaps in timestamps. The package now compares timestamps to the nearest `second` when selecting valid SLA policy names in [int_zendesk__sla_policy_applied](https://github.com/fivetran/dbt_zendesk/blob/main/models/sla_policy/int_zendesk__sla_policy_applied.sql).
+
+## Under the Hood
+- Updated `consistency_sla_policies` data validation test to account for the above change.
```

Comment:
We can probably combine the two sentences here into one, I think they were both to account for the above changes?

Reply:
combined